### ISSUE TITLE
Performance: attic create stalls while index is read back

### ISSUE THREAD

**Issue by tscheggsches** [jborg/attic#307]
Hi there,

Thank you for attic! I back up up around 80GB over sshfs. On regular intervals (creating checkpoints?) I noticed a stall on my WLAN link while sending data. I guess this is while the index is read back (guess supported by lsof -c attic). As the archive grows, the stalls gets longer. In my case, the index is around 80MB and takes a while over a slow internet link.

I am wondering why an index has ever to be read back? I would think an index could be written simultaneously locally and transfered to the remote side?

**Comment by ThomasWaldmann** [comment]
Just in case you didn't read it already: try ssh (+ attic on remote side), could be more efficient.

**Comment by tscheggsches** [comment]
Good idea, but the NAS I access (LACIE Cloudbox) only supports sftp ;). And why should my observed behaviour change anyway?

**Comment by ThomasWaldmann** [comment]
Because you then have a remote process that can work with the repo data on the remote side. Avoiding to pump some stuff over the network connection. With a just a mounted network filesystem, you have no other option...

**Comment by ThomasWaldmann** [comment]
3 potential remedies for this:
- use a remote attic process (ssh: repo url - not always possible)
- do checkpoints less often (already possible: use `attic create --checkpoint-interval NNNN`)
- have less chunks / smaller index files (chunk size not configurable yet)
